patches/minecraft/net/minecraft/client/renderer/LevelRenderer.java.patch

```diff
@@ -89,7 +90,18 @@
        }
  
        PoseStack posestack = RenderSystem.m_157191_();
-@@ -1344,7 +_,7 @@
+@@ -1337,14 +_,18 @@
+          this.f_109413_.m_83954_(Minecraft.f_91002_);
+          this.f_109413_.m_83945_(this.f_109461_.m_91385_());
+          profilerfiller.m_6182_("translucent");
++         if (!net.minecraftforge.client.renderer.LevelRendererAdapter.getInstance().onRenderTranslucent(this, p_109600_, p_109604_, p_109607_)) {
+          this.m_172993_(RenderType.m_110466_(), p_109600_, d0, d1, d2, p_109607_);
```

Comment:
Looking at the code in `renderChunkLayer` shows some hardcoded checks for `RenderType.translucent()`. Do these checks substantially affect render types which render alongside the vanilla translucent render type?

We may want to expand the test mod to include tests for these three render type 'groups', rather than only registering a custom solid render type.